docs/pages/develop/user-interface/app-icons.mdx

```diff
@@ -30,16 +30,16 @@ For an in-detail walkthrough, see the video below:
 
 Further customization of the Android icon is possible using the [`android.adaptiveIcon`](/versions/latest/config/app/#adaptiveicon) property, which will override both of the previously mentioned settings.
 
-The Android Adaptive Icon is formed from two separate layers &mdash; a foreground image and a background color or image. This allows the OS to mask the icon into different shapes and also supports visual effects. For Android 13 and later, the OS supports a monochrome foreground image layer which will use a background color taken from the device's theme.
+The Android Adaptive Icon is formed from two separate layers &mdash; a foreground image and a background color or image. This allows the OS to mask the icon into different shapes and also supports visual effects. For Android 13 and later, the OS supports a themed app icon that uses a wallpaper and theme to determine the color set by the device's theme.
 
 The design you provide should follow the [Android Adaptive Icon Guidelines](https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive) for launcher icons. You should also:
 
 - Use **.png** files.
-- Use the `android.adaptiveIcon.foregroundImage` field in **app.json** to specify path to your foreground image.
-- Use the `android.adaptiveIcon.monochromeImage` field in **app.json** to specify path to your monochrome image.
-- The default background color is white; to specify a different background color, use the `android.adaptiveIcon.backgroundColor` field. You can instead specify a background image using the `android.adaptiveIcon.backgroundImage` field. Make sure that it has the same dimensions as your foreground image.
+- Use the `android.adaptiveIcon.foregroundImage` property to specify the path to your foreground image.
+- Use the `android.adaptiveIcon.monochromeImage` property to specify the path to your themed app icon image.
```

Comment:
Small nit, but I would change this back to `monochrome image`. As this is the monochrome image your provide to be able to use the themed app icon feature.